docs/docs/introduction/container-images.md

```diff
@@ -0,0 +1,199 @@
+---
+layout: default
+title: Container Images
+nav_order: 2
+parent: Introduction
+---
+
+# Introduction to Containers
+
+A container is a portable encapsulation of an environment. It's a
+way for you to package an entire operating system, libraries, and
+dependencies in a package that you can move and run anywhere.
+That is the quick and dirty definition, but today we are going
+to delive a little more into what this actually means.
+
+{: .no_toc }
+
+## Table of contents
+{: .no_toc .text-delta }
+
+1. TOC
+{:toc}
+
+---
+
+## Container Basics
+
+### What problems do containers solve?
+
+
+#### Reproducibility
+{: .no_toc }
+
+We have differences in environments, hardware, and even security, and this makes sharing of applications challenging, leading to the commonality of the phrase above. A *reproducible* workflow that uses containers helps us get around this problem, and (hopefully) reduce the frequency of uttering this phrase _But it ran on my computer!_
+
+#### Portability
+{: .no_toc }
+
+The ability to easily move a container from one machine to another is easily afforded by the fact that many container formats are variations on a compressed filesystem. If you wanted to share your entire computer, right now, you would have a hard time doing that.
+
+#### Development Environments
+{: .no_toc }
+
+It used to be the case that to develop a simple model - view - control application with some application, database, and web server, you would need to install and run the components on your host. It would be very challenging to work on multiple projects at once. Containers are essential for developers because 
+they allow us to spin up an entire set of components for a specific development environment, and then bring it
+down just as easily.
+
+#### Continuous Integration
+{: .no_toc }
+
+The same is true for testing, You can easily test across operating systems without leaving a single host.
```

Comment:
/s/You can easily test across operating systems without leaving a single host./using containers one can easily test across operating systems from a single host./